packages/im-components/src/molecules/logo/Logo.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Link } from "react-router-dom";
+
+// Components
+import { VideoCameraFilled } from "imcomponents/atoms/icon";
+
+// Styles
+import styles from "./logo.module.scss";
+
+// Constants
+import { EMPTY_STRING } from "imbase/constants/base.constants";
+
+const Logo = ( props ) => {
+  const { text, children } = props
+  return (
+    <>
+      <Link to={"/"} className={styles.logo} >
+        <VideoCameraFilled className={styles.image} style={{ fontSize:"250%" }} />
+        <div className={styles.text} >{text}</div>
+      </Link>
+    </>
+  );
+}
+
+Logo.propTypes = {
+  text: PropTypes.string
```

Comment:
children should be props

Reply:
Will pull changes from develop once #18 is merged. updated it there.